Description: Sets normalize-audio as the default normcmd in /etc/ripit/config.
Author: Elimar Riesebieter <riesebie@lxtec.de>
Last-Update: 2010-02-21

Index: ripit-3.9.0/config
===================================================================
--- ripit-3.9.0.orig/config	2010-09-25 17:56:29.552477937 +0200
+++ ripit-3.9.0/config	2010-09-25 17:56:40.368725615 +0200
@@ -875,9 +875,10 @@
 # normcmd: Command to be used to normalize.
 # Possible values: string
 # Example: normalize-audio (when using Debian)
-# Default: normalize
+# Upstream default: normalize
 
-normcmd=normalize
+# We are running Debian so set to normalize-audio
+normcmd=normalize-audio
 
 # normopt: Options to pass to normalize
 # Possible values: -a -nndB   : Normalize to -nn dB, default is -12dB,
